lib/warnings.pm

```diff
@@ -689,6 +689,10 @@ disable compile-time warnings you need to rewrite the code like this:
 	 my $b; chop $b;
      }
 
+And note that unlike the first example, this will permanently set $^W
```

Comment:
Shouldn't that be wrapped with C<>, like the rest of the pod

Reply:
Fixed